tools/fastg2protlib/fastg2protlib-peptides.xml

```diff
@@ -2,21 +2,19 @@
     <macros>
         <import>macros.xml</import>
     </macros>
-    <description>Generate a peptide FASTA library starting with a FASTG assembly file.</description>
-
+    <description>Generate FASTA from FASTG.</description>
     <requirements>
```

Comment:
Just put the whole `<requirements>` in the macro, not just `<requirement>`

Reply:
Done